sys/net/routing/rpl/rpl.c

```diff
@@ -116,6 +110,11 @@ uint8_t rpl_init(int if_id, ipv6_addr_t *address)
     return SIXLOWERROR_SUCCESS;
 }
 
+kernel_pid_t rpl_get_if(void)
```

Comment:
Moreover: which pid is returned here? The interface representing RPL? Or the interface RPL is registered to (as in "is a router for"). The first case doesn't make sense, since interfaces are protocol threads that IP sends stuff over. The second case disregards the possibility to have RPL on more than one interface. Solution could be to give RPL its own interface representation (similar to `ng_ipv6_netif`).